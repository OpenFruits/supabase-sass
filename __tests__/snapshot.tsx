import renderer from "react-test-renderer";

import About from "../pages/about";
import Index from "../pages/index";

it("renders index unchanged", () => {
  const tree = renderer.create(<Index />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders about unchanged", () => {
  const tree = renderer.create(<About />).toJSON();
  expect(tree).toMatchSnapshot();
});
