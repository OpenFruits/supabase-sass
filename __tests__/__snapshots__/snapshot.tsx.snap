// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders about unchanged 1`] = `
<div>
  <div>
    <h2>
      About
    </h2>
  </div>
</div>
`;

exports[`renders index unchanged 1`] = `
<div>
  <div>
    <h2>
      Index
    </h2>
    <button
      className="p-2"
      onClick={[Function]}
    >
      show alert!
    </button>
  </div>
</div>
`;
